index.html

```diff
@@ -33,5 +33,183 @@ <h2 class="social">LET'S CONNECT</h2>
+                                <img src="./img/Counter.png" alt="" class="counter">
+                                <span class="year">2015</span>
+                            </div>
+                        </div>
+                    </div>
+                    <span class="details">
+                            A daily selection of privately personalized reads; no accounts or sign-ups required.
+                    </span>
+                    <ul class="tags">
+                        <li>html</li>
+                        <li>css</li>
+                        <li>javaScript</li>
+                    </ul>
+                    <button class="action-button">See Project</button>
+                </div>
+    
+                <div class="card">
+                    <img src="./img/Snapshoot Portfolio (2).png" alt="snapshot poortfolio" class="snapshot">
+                    <div class="works-description">
+                        <div class="works-intro">
+                            <h1 class="project-title">Tonic</h1>
+                            <div class="frame">
+                                <span class="client">
+                                    CANOPY</span>
+                                <img src="./img/Counter.png" alt="" class="counter">
+                                <span class="role">Back End Dev</span>
+                                <img src="./img/Counter.png" alt="" class="counter">
+                                <span class="year">2015</span>
+                            </div>
+                        </div>
+                    </div>
+                    <span class="details">
+                            A daily selection of privately personalized reads; no accounts or sign-ups required.
+                    </span>
+                    <ul class="tags">
+                        <li>html</li>
+                        <li>css</li>
+                        <li>javaScript</li>
+                    </ul>
+                    <button class="action-button">See Project</button>
+                </div>
+    
+                <div class="card">
+                    <img src="./img/Snapshoot Portfolio (3).png" alt="snapshot poortfolio" class="snapshot">
+                    <div class="works-description">
+                        <div class="works-intro">
+                            <h1 class="project-title">Multi-Post
+                                Stories</h1>
+                            <div class="frame">
+                                <span class="client">
+                                    CANOPY</span>
+                                <img src="./img/Counter.png" alt="" class="counter">
+                                <span class="role">Back End Dev</span>
+                                <img src="./img/Counter.png" alt="" class="counter">
+                                <span class="year">2015</span>
+                            </div>
+                        </div>
+                    </div>
```

Comment:
- [ ] Kindly follow the design in Figma as much as possible. As you can see in the template the texts are not bordered 👍 

## Your design:
![image](https://user-images.githubusercontent.com/50721479/205450862-e451bd70-8b82-45c6-9472-b1c28b6a0a70.png)

## Figma design:
![image](https://user-images.githubusercontent.com/50721479/205450872-fbf7e5b7-e8ae-4f54-856d-cf1adf512e2e.png)